src/Keter/Plugin/Postgres.hs

```diff
@@ -4,6 +4,7 @@ module Keter.Plugin.Postgres
     ( -- * Settings
       Settings
     , setupDBInfo
+    , defaultSettings
```

Comment:
I think we should consider all these new default functions now. We've already broke backwards compatibility but for a nice upgrade path it's  better if we expose all default configuration, since `data.default` is public by default.